Dbal/PaymentAttributes.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AdyenPayment\Dbal;
+
+use Enlight_Components_Db_Adapter_Pdo_Mysql;
+
+class PaymentAttributes
+{
+    /**
+     * @var Enlight_Components_Db_Adapter_Pdo_Mysql
+     */
+    private $db;
+
+    public function __construct(Enlight_Components_Db_Adapter_Pdo_Mysql $db)
+    {
+        $this->db = $db;
+    }
+
+    public function fetchByAdyenType(string $adyenType): array
+    {
+        $paymentAttributesResult = $this->db
+            ->select()
+            ->from('s_core_paymentmeans_attributes')
+            ->where('adyen_type=?', $adyenType)
```

Comment:
What happens if an empty ` $adyenType` is provided?
Better to handle those edge case as it could result in unexpected behaviour.